docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -2,47 +2,134 @@
+Metabase will start using the default settings. You should see some log entries starting to run in your terminal window showing you the application progress as it starts up. Once Metabase is fully started you'll see a confirmation such as:
 
-    ...
-    06-19 10:29:34 INFO metabase.task :: Initializing task CheckForNewVersions
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendAnonymousUsageStats
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendAbandomentEmails
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendPulses
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendFollowUpEmails
-    06-19 10:29:34 INFO metabase.task :: Initializing task TaskHistoryCleanup
-    06-19 10:29:34 INFO metabase.core :: Metabase Initialization COMPLETE
+```
+...
+06-19 10:29:34 INFO metabase.task :: Initializing task CheckForNewVersions
+06-19 10:29:34 INFO metabase.task :: Initializing task SendAnonymousUsageStats
+06-19 10:29:34 INFO metabase.task :: Initializing task SendAbandomentEmails
+06-19 10:29:34 INFO metabase.task :: Initializing task SendPulses
+06-19 10:29:34 INFO metabase.task :: Initializing task SendFollowUpEmails
+06-19 10:29:34 INFO metabase.task :: Initializing task TaskHistoryCleanup
+06-19 10:29:34 INFO metabase.core :: Metabase Initialization COMPLETE
+```
 
 At this point you're ready to go! You can access your new Metabase server on port 3000, most likely at [http://localhost:3000](http://localhost:3000)
 
 You can use another port than 3000 by setting the `MB_JETTY_PORT` [environment variable](./environment-variables.md) before running the jar.
 
-Note that in the default configuration Metabase will use a local H2 database for storing all its own application data. This is meant for simple evaluations or personal use, so if you want to run Metabase in production we recommend you [migrate away from H2](./migrating-from-h2.md).
+Note that in the default configuration Metabase will use a local H2 database for storing all its own application data. This default is meant for simple evaluation or personal use. If you want to run Metabase in production we recommend you [migrate away from H2](./migrating-from-h2.md).
+
+## Production installation
+
+The steps are the same as above with one important difference: if you want to run Metabase in production, you'll want to use a production-ready database to store your Metabase application data. Here are some [databases we recommend](migrating-from-h2.md#databases-we-recommend-for-storing-your-metabase-application-data).
+
+For example, say you want to use [PostgreSQL](https://www.postgresql.org/). You would get a PostgreSQL service up and running and create an empty database:
+
+```
+createdb metabaseappdb
```

Comment:
Is this correct? I always get into the DB with the psql client and run "CREATE DATABASE metabase;"

Reply:
Yes, it's a command: https://www.postgresql.org/docs/14/app-createdb.html
You can use SQL (or PgAdmin or whatever tool you prefer) to create the database.